src/tests/core.py

```diff
@@ -274,6 +276,52 @@ def test_old_knownnodes_pickle(self):
         finally:
             cleanup(files=('knownnodes.dat',))
 
+    @staticmethod
+    def generate_random_address():
+        """Generating random address"""
+        import queues
+        streamNumberForAddress = 1
+        queues.addressGeneratorQueue.put((
+            'createRandomAddress', 4, streamNumberForAddress,
+            "test1", 1, "", False))
+
+    @staticmethod
+    def delete_address_from_database(address):
```

Comment:
this method should be called `delete_address_from_addressbook` to avoid conflusion